packages/io/io/README.md

```diff
@@ -26,7 +26,6 @@ This package is a metapackage and includes all the input/output format handling
 ie: file data => JSCAD code or JSCAD geometry
 
 Following formats are supported as inputs
-- [AMF](../amf-deserializer)
```

Comment:
Why no AMF? I know it's not very popular. But it is meant to be a "better" STL file. And the converter is already written. I'm fine with removal, just curious.

Reply:
My knowledge is limited about AMF, but I have heard from dude that worked on 3mf, that AMF failed in the sense to deliver on those promises in practice. Still I am also curious why remove it ?

Reply:
> Why no AMF? I know it's not very popular. But it is meant to be a "better" STL file. And the converter is already written. I'm fine with removal, just curious.

The simple answer is "maintenance". I don't want to waste time and effort maintaining this further. We have done this for other formats (gcode, scad) so there's nothing new here.

FYI, I will be removing gcode and scad from the V3 branch. If someone wants these then let them port V1 to V3, with tests suites as well.